hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java

```diff
@@ -169,7 +170,12 @@ protected boolean submitOp(HapiSpec spec) throws Throwable {
                             "{} Status resolution failed due to unrecoverable runtime exception, "
                                     + "possibly network connection lost.",
                             TxnUtils.toReadableString(txn));
-                    throw new HapiTxnCheckStateException("Unable to resolve txn status!");
+                    if (unavailableStatusIsOk) {
+                        // If we expect the status to be unavailable treat this as a success
+                        return true;
```

Comment:
Is this for some test to pass ?

Reply:
Yep, elaborated in the comment here; allows tests that submit transactions > 6144 bytes to pass.